Mailbox/Person.cs

```diff
@@ -1,4 +1,33 @@
-﻿namespace Mailbox
+﻿using System;
+using System.Diagnostics.CodeAnalysis;
+
+namespace Mailbox
 {
+    public struct Person : IEquatable<Person>
+    {
+        public string _FirstName; //{ get; set; }
+        public string _LastName; //{ get; set; }
+
+        public Person(string _FirstName, string _LastName)
```

Comment:
In member Mailbox.Person.Person(string, string), remove the underscores from parameter name _FirstName.
In member Mailbox.Person.Person(string, string), remove the underscores from parameter name _LastName.

```suggestion
        public Person(string firstName, string lastName)
```